src/Cpanel/ClientActions.php

```diff
@@ -143,15 +143,14 @@ public function addSSLVerficiationDNSRecordForCName($zoneList)
                         $dnsRecord->setContent($recordTarget);
                         $dnsRecord->setTtl(14400);
 
-                        return $this->cpanelAPI->addDNSRecord($zoneName, $dnsRecord);
+                        $this->cpanelAPI->addDNSRecord($zoneName, $dnsRecord);
```

Comment:
Why was this removed?  This is how you tell whether the record was added successfully or not.